t/admin/routes.t

```diff
@@ -321,6 +328,7 @@ GET /t
                 }]]
                 )
 
+            ngx.header["Content Type"] = res.header["Content Type"]
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/85495012-529b6580-b60c-11ea-80c0-12aac7e929b4.png)

Reply:
I think this issue has been already resolved in Openresty